Aleph1.Skeletons.WebAPI/Aleph1.Skeletons.WebAPI.Pages/src/resources/helpers/format-birthdate.ts

```diff
@@ -0,0 +1,17 @@
+import { format, parseISO } from "date-fns";
+
+export function formatBirthdate(birthdate: Date | string): string
```

Comment:
`format-birthdate` might be the wrong name when thinking about it. Maybe it should be named `format-date`?
I chose to use a specific function for it because it is the most used and sought after format when dealing with dates.

Also, handling string edge cases for when the function is not used on Dates, you never know...